src/promptflow-devkit/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -1121,7 +1119,46 @@ def infer_signature(self, entry: Callable) -> dict:
         :rtype: dict
         """
         # TODO: should we support string entry? If so, we should also add a parameter to specify the working directory
-        flow_meta, _, _ = self._infer_signature(entry=entry)
+        if isinstance(entry, Prompty):
```

Comment:
putting implementation to `_infer_signature` is to avoid creating public call telemetry. Please move below logic to `_infer_signature`

Reply:
mvoe to _infer_signature